cmd/check/main.go

```diff
@@ -32,12 +33,27 @@ func main() {
 		log.Fatalf("failed unmarshalling: %s", err)
 	}
 
-	client := boshio.NewClient(nil, nil)
+	client := boshio.NewClient(nil, nil, checkRequest.Source.ForceRegular)
 	stemcells, err := client.GetStemcells(checkRequest.Source.Name)
 	if err != nil {
 		log.Fatalf("failed getting stemcell: %s", err)
 	}
 
+	supportsLight := client.SupportsLight(stemcells)
+	if supportsLight {
+		if checkRequest.Source.ForceRegular {
+			regularFilter := func(stemcell boshio.Stemcell) bool {
+				return stemcell.Regular != nil
+			}
+			stemcells = client.FilterStemcells(regularFilter, stemcells)
+		} else {
+			lightFilter := func(stemcell boshio.Stemcell) bool {
+				return stemcell.Light != nil
```

Comment:
Hmm, should this instead be a no-op (`return true`)? It seems like it'd exclusively only match stemcells with a light version available. What if it's an IaaS that only ever has regular stemcells and `force_regular` is false?